README.md

```diff
@@ -35,6 +35,7 @@ For a longer list, see:
 For a list of free services and tools for open source .NET projects, see:
 
 * [Free Services & Tools for Open Source .NET Projects](dotnet-free-oss-services.md)
+* [Download the latest Azure .NET SDK](https://www.visualstudio.com/features/azure-tools-vs)
```

Comment:
I have a different thread with Lee Coward on this. It is not open source projects, but it is free tools and services to help on any .NET projects.